packages/react-mutation-mapper/src/component/civic/CivicCard.tsx

```diff
@@ -24,7 +26,7 @@ export default class CivicCard extends React.Component<ICivicCardProps, {}> {
 
         if (variantMap) {
```

Comment:
i think we don't needs this outside check because any falsy state of variantMap will be handled by _.isEmpty